### DISCUSSION TITLE
Does simplelogin.io have an onion address

### DISCUSSION THREAD

**Discussion by Night3890** [simple-login/app#1343] [Q&A] [answered]
Hello, 

Does simplelogin.io have a .onion address? If not, when will it get one like [proton.me](https://account.protonmailrmez3lotccipshtkleegetolb73fuirgj7r4o4vfu7ozyd.onion/login) has?

Thank you

**Comment by nguyenkims** [comment, marked as answer]
Currently SL doesn't have an onion address. This is a good idea indeed 👍

**Reply by Night3890** [reply]
Any idea when we can expect one? I figured it would be soon since you SL was bought by proton but i don't see any indications of it happening.

**Comment by ghost** [comment]
I support this. @nguyenkims proton already supports it. Can you please do it too?
